app/src/bcsc-theme/features/verify/VerifyIdentityStack.tsx

```diff
@@ -103,6 +108,31 @@ const VerifyIdentityStack = () => {
       <Stack.Screen name={BCSCScreens.TransferAccountInstructions} component={TransferInstructionsScreen} />
       <Stack.Screen name={BCSCScreens.TransferAccountQRScan} component={TransferQRScannerScreen} />
       <Stack.Screen name={BCSCScreens.IdentitySelection} component={IdentitySelectionScreen} />
+      <Stack.Screen
+        name={BCSCScreens.EditNickname}
+        component={EditNicknameScreen}
+        options={{ title: t('BCSCSettings.EditNickname') }}
+      />
+      <Stack.Screen
+        name={BCSCScreens.ForgetAllPairings}
+        component={ForgetAllPairingsScreen}
+        options={{ title: t('BCSCSettings.ForgetPairings') }}
```

Comment:
https://www.figma.com/design/LaNyNJGxNMsxlqUCNTWy1X/BCSC-V4---Phase-1?node-id=432-432&m=dev

Looking at the wireframes, I don't think these two are supposed to show a title